tests/test_ext_imgconverter.py

```diff
@@ -1,12 +1,26 @@
 """Test sphinx.ext.imgconverter extension."""
 
-import os
+import subprocess
+from subprocess import PIPE
 
 import pytest
 
 
+@pytest.fixture
+def if_converter_found(app):
+    image_converter = getattr(app.config, 'image_converter', '')
```

Comment:
I don't think `sphinx.ext.imgconverter` is loaded automatically, so this would always be `''`, right? (Unless `app` picks up the `testroot` from the below `mark.sphinx`, but I don't know that much about how PyTest works)

A

Reply:
On my local and on GitHub Actions, it works fine.
https://github.com/sphinx-doc/sphinx/runs/6260118909?check_suite_focus=true#step:9:1279